homework.md

```diff
@@ -44,37 +44,38 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
-
+A programming language used to add complex features on web pages 
```

Comment:
On the contrary JavaScript is more than just a programming language for **making complex features on web pages** and answering the question the way you did, is for another question let me explain myself.

So JavaScript ever since 2009 when NodeJS was introduced one could now build web services and CLI's with JavaScript, come 2015 and React Native was introduced and you could now create Native Mobile Applications with JavaScript and finally 2019 came ElectronJS one can now build Desktop Applications with JavaScript...
So my point here is JavaScript is more than what you say it is plus you basically answering "what JavaScript does" not "what JavaScript is".

### The key here is you are to explain what is JavaScript, not  what it does

Reply:
Thank you Ntshembo for pointing me in the right direction. I will add your input. Ps. My answer is what I recalled from memory. I wanted to leave google out of the homework.